### ISSUE TITLE
Website appears down

### ISSUE THREAD

**Issue by gnerdalert** [lyrgard/ffbeEquip#509]
When attempting to go on to the site now, I just get "The site cannot be reached." Has the site been taken down purposefully or is there something broken?

**Comment by lyrgard** [comment]
Yes, my server crashed down. I created a fallback site in the time being, until I'm able to put ffbe equip back online : https://ffbeequip.uc.r.appspot.com/
